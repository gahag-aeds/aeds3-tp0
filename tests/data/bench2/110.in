110 55
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 29 91 71 83 57 73 47 25 2 90 
Add 55 90
Min 22 57
Add 33 68
Max 24 59
Sub 40 75
Max 36 71
Sub 10 45
Min 33 68
Sub 58 93
Sum 39 74
Sub 72 107
Sum 3 38
Sub 60 95
Min 13 48
Add 63 98
Sum 47 82
Add 19 54
Min 3 38
Add 63 98
Min 40 75
Add 57 92
Sum 28 63
Add 23 58
Max 7 42
Add 9 44
Sum 12 47
Sub 44 79
Max 40 75
Add 67 102
Sum 37 72
Sub 42 77
Max 48 83
Sub 36 71
Sum 52 87
Add 49 84
Min 53 88
Add 13 48
Max 58 93
Add 11 46
Sum 49 84
Sub 22 57
Max 60 95
Add 51 86
Sum 60 95
Sub 50 85
Min 17 52
Sub 32 67
Sum 68 103
Add 19 54
Min 72 107
Add 11 46
Min 54 89
Add 25 60
Min 33 68
Sub 60 95
