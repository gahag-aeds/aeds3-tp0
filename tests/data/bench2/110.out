1
96
99
1
1458
1772
1
1715
7
1
1506
95
1713
97
1510
96
1693
2
100
1698
100
1995
1
2158
2
0
1
